src/content/blog/val-town-newsletter-25.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Val Town Newsletter 25
+description: TODO
+pubDate: March 19, 2025
+author: Steve Krouse
+---
+
+import { Image } from "astro:assets";
+
+## Edit Val Town Projects Locally
+- vt
+
+TODO
+
+## Val Town Projects Update
+- Forks are now Remixes
+- Projects API - Write + Delete
+- Customizable `val.run` domains for Project HTTP vals (soon)
+
+## Logs
+- Raw logs + traces
+- Account-wide logs (visit val.town/u/yourusername/logs)
+
+## Other updates
+- Updated profile conslidating projects and vals
+- Support for Bluesky links in profile
+- Backend support for commit messages (front end missing at time of writing)
+- Browse file history at any snapshot in time
+- PRs now support title and markdown descriptions
+- Minor footer redesign with prettier social icons
+- Support formatting HTML, CSS, JSON file types 
+- API Token page now loads significantly faster
+- New Project Templates (ft. https://www.val.town/x/fiberplane/fiberplaneHonoZodStarter)
+- Added /llms.txt and /llms-full.txt to docs page, can copy any page as markdown for LLM
+
+## Roadmap
```

Comment:
i'm probably going to redo this section fyi

Reply:
Oh yes please feel free to do anything with this btw, I was just dumping stuff on the page to start